lightning/src/ln/channel.rs

```diff
@@ -1182,6 +1181,30 @@ impl UnfundedChannelContext {
 	}
 }
 
+/// Info about a pending splice, used in the pre-splice channel
+#[cfg(splicing)]
+#[derive(Clone)]
+struct PendingSpliceInfoPre {
```

Comment:
Trying to think of a better name for this. Is it necessary to divide this into "pre" and "post" structs? Thinking we can simply call this `PendingSplice`.

Reply:
Renamed